net/http/http_network_session.cc

```diff
@@ -418,6 +418,18 @@ CommonConnectJobParams HttpNetworkSession::CreateCommonConnectJobParams(
       for_websockets ? &websocket_endpoint_lock_manager_ : nullptr);
 }
 
+#if defined(STARBOARD)
```

Comment:
We should replace all instances of `#if defined(STARBOARD)` in this PR with `#if defined(USE_COBALT_CUSTOMIZATIONS)`